index.html

```diff
@@ -31,21 +31,21 @@
       <div class="row">
         <div class="col-md-4 col-md-offset-4">
           <ul class="list-inline social-buttons">
-              <li>
-                  <a target="_blank" href="https://www.facebook.com/gutechsoc"><i class="fab fa-facebook-f"></i></a>
-              </li>
-              <li>
-                  <a target="_blank" href="https://twitter.com/gutechsoc"><i class="fab fa-twitter"></i></a>
-              </li>
-              <li>
-                <a target="_blank" href="https://www.youtube.com/channel/UCbwnhXjLXAWxc6Nj_z-tlyQ/videos?view=57&flow=grid"><i class="fab fa-youtube"></i></a>
-              </li>
-              <li>
-                  <a target="_blank" href="https://discord.gg/EGtY9n"><i class="fab fa-discord"></i></a>
-              </li>
-              <li>
-                  <a target="_blank" href="mailto:gutechsoc@gmail.com"><i class="fas fa-envelope"></i></a>
-              </li>
+            <li>
+              <a target="_blank" href="https://www.facebook.com/gutechsoc"><i class="fab fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a target="_blank" href="https://twitter.com/gutechsoc"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li>
+              <a target="_blank" href="https://discord.gg/EGtY9n"><i class="fab fa-discord"></i></a>
+            </li>
+            <li>
+              <a target="_blank" href="https://www.youtube.com/channel/UCbwnhXjLXAWxc6Nj_z-tlyQ/videos?view=57&flow=grid"><i class="fab fa-youtube"></i></a>
+            </li>
+            <li>
+              <a target="_blank" href="https://www.linkedin.com/company/glasgow-university-tech-society?trk=organization-update_share-update_actor-text"><i class="fab fa-linkedin"></i></i></a>
```

Comment:
```suggestion
              <a target="_blank" href="https://www.linkedin.com/company/glasgow-university-tech-society"><i class="fab fa-linkedin"></i></i></a>
```

Reply:
The option seems unnessesary

Reply:
That's true. I have committed the suggested change now. Thank you for spotting this!